Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
> getDefaultOptions has some differences.

I guess this is mostly for historical reasons, i.e. changes not being systematically ported to all bundles.
The phpcr admin bundle is the only one having meaningful differences.

That's why I proposed an [`AbstractFormContractor`](https://github.com/sonata-project/SonataAdminBundle/pull/3971/files#diff-672268a470459c46dc5898c5f7265f6cR20) class in #3971